File: app/src/main/java/com/example/simeons/projectmanager/Model/WorkLog.java
package com.example.simeons.projectmanager.Model;

/**
 * Created by SimeonS on 18/11/2016.
 */

public class WorkLog {

    public String ID;
    public String TaskID;
    public String PersonID;
    public String Person;
    public String Time;
    public String DateCreated;

    public WorkLog(String iD, String taskID, String personID, String person, String time, String dateCreated){

        this.ID = iD;
        this.TaskID = taskID;
        this.Person = person;
        this.PersonID = personID;
        this.Time = time;
        this.DateCreated = dateCreated;

    }

}

File: ProjectManagerAPI/ProjectManagerAPI/Models/WorkLog.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Web;

namespace ProjectManagerAPI.Models
{
    public class WorkLog
    {
        public int? ID { get; set; }
        public int? TaskID { get; set; }
        public string Person { get; set; }
        public int? Time { get; set; }
    }
}